src/components/Login/Login.jsx

```diff
@@ -5,7 +5,14 @@ import './Login.css';
 const Login = () => {
   return (
     <div className='Login'>
-      Login Placeholder
+      <h2 className='greeting'>
+        Welcome Traveler...
+      </h2>
+      <div className='login-container'>
+        <button className='login-button'>user1</button>
+        <button className='login-button'>user2</button>
+        <button className='login-button'>user3</button>
```

Comment:
I might consider naming the users something a little more descriptive regarding their roles or how they're different. If there are three, there is probably a reason?

Reply:
Good callout! I figured I would update them once we had the data in place, since I'm not entirely sure which will have what until we have that data.

Reply:
If it helps, how I created our seed data, it is for a user without any encounters set up, a 'standard' one with a few, and another with many set. (thought these three cases would help with the varying cases of display to help test)

Reply:
I'll go ahead and update them now, based on our slack convo.
- User with NO existing encounters
- User with ONE existing encounter
- User with MANY existing encounters